tests/backup/backup_test.go

```diff
@@ -398,7 +398,6 @@ var _ = Describe("{BasicBackupCreation}", func() {
 	JustBeforeEach(func() {
 		backupLocationMap = make(map[string]string)
 		labelSelectors = make(map[string]string)
-		bkpNamespaces = make([]string, 0)
```

Comment:
Wont this fail to compile if you remove the declaration. When you use `bkpNamespaces`, will it not be undefined?

Reply:
The var keyword creates an uninitialized slice with a nil value. Then, the append function will create a new slice as it is nil initially and appends elements to the slice. The test case has passed. I am considering we should use make only when necessary and let all the variables be nil initially. This will help avoid confusion.

Reply:
Added it back.